lsp/src/htmx/mod.rs

```diff
@@ -220,6 +225,10 @@ pub fn init_hx_tags() {
         ("hx-swap-oob", include_str!("./attributes/hx-swap-oob.md")),
         ("hx-confirm", include_str!("./attributes/hx-confirm.md")),
         ("hx-disable", include_str!("./attributes/hx-disable.md")),
+        (
+            "hx-disabled-elt",
+            include_str!("./attributes/hx-disabled-elt.md"),
+        ),
```

Comment:
It might be nicer if this has the same formatting as the other elements of the array

Reply:
It might be due to a linear length constraint in an autoformatter.